jobs/config_test.py

```diff
@@ -373,6 +388,23 @@ def get_real_bootstrap_job(self, job):
         self.assertIn(key, sorted(self.realjobs))  # sorted for clearer error message
         return self.realjobs.get(key)
 
+    def test_non_blocking_jenkins(self):
+        """All PR non-blocking jenkins jobs are always_run: false"""
+        # ref https://github.com/kubernetes/test-infra/issues/4637
+        if not self.presubmits:
+            self.load_prow_yaml(self.prow_config)
+        required_jobs = get_required_jobs()
+        message_format = 'Jenkins jobs should not be `always_run: true`' + \
```

Comment:
nit: just include the message format in the `self.fail` call?

Reply:
done